tests/unittests/test_call_endpoint.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
```

Comment:
[NIT] If this can not be executed as a file, do not add the shebang.

Reply:
Done